package.json

```diff
@@ -83,7 +85,7 @@
       "<rootDir>/packages/core/"
     ],
     "testMatch": [
-      "<rootDir>/packages/core/src/**/__tests__/?(*.)spec.ts(x|)"
+      "<rootDir>/packages/core/src/**/?(*.)test.ts(x|)"
```

Comment:
As with the previous PR, let's maintain `spec` for the naming convention, and `__tests__` for the containing folder :-)

Reply:
Let's catch up about this, I'd love to maintain Candu's convention of `tests` and `test`. There are effectively no unit tests in Craft now, so let's maintain existing conventions.